src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -398,6 +399,9 @@ def report_metrics_for_detectors(
         if detector.type in [DetectorType.UNCOMPRESSED_ASSETS]:
             detected_tags["browser_name"] = allowed_browser_name
 
+        if detector.type in [DetectorType.CONSECUTIVE_HTTP_OP]:
+            detected_tags["frontend"] = is_event_from_browser_javascript_sdk(event)
```

Comment:
it's a bool so maybe this would be a bit better
```suggestion
            detected_tags["is_frontend"] = is_event_from_browser_javascript_sdk(event)
```